src/components/api-calls/Fetch.js

```diff
@@ -0,0 +1,46 @@
+/**
+ * Fetch from http://localhost:5000/graphql without auth header
+ * @param {Object} requestBody defines query
+ * @returns {Promise} response
+ */
+export const fetchNoAuth = (requestBody) => {
+  return fetch('http://localhost:5000/graphql', {
+    method: 'POST',
+    body: JSON.stringify(requestBody),
+    headers: {
+      'Content-Type': 'application/json'
+    }
+  })
+    .then(res => {
+      if (res.status !== 200 && res.status !== 201) {
+        alert("Something went wrong.");
+        throw new Error('Failed');
+      }
+      return res.json();
+    });
+};
+
+/**
+ * Fetch from http://localhost:5000/graphql with auth header
+ * 
+ * @param {string} token
+ * @param {Object} requestBody defines query
+ * @returns {Promise} response
+ */
+export const fetchAuth = (token, requestBody) => {
+  return fetch('http://localhost:5000/graphql', {
```

Comment:
Is it always localhost:5000? Could it by any chance change?

Reply:
Hi @domhanak ,
yes it can change after deployment.
Is it ok if I fix this in the next pull request with deployment-ready changes?